airflow/opt/providers/etna/etna/dags/decorators.py

```diff
@@ -49,19 +49,37 @@ def instantiate_dag(fn):
             start_date=start_date,
             end_date=end_date,
             schedule_interval=schedule_interval,
+            max_active_runs=1,
             **kwds,
         ) as dag:
             inject(fn, inject_params)
         return dag
 
     return instantiate_dag
 
+def rollup_dag(interval: timedelta, labels: List[str]) -> Callable[[Callable], DAG]:
+    """
+    `system_dag` variant that sets the dag's labels so that a rollup of it's XCom values are used
+    """
+
+    def instantiate_dag(fn):
+        return dag(
+            start_date=system_epoch,
+            schedule_interval=interval,
+            default_args=dict(
+                owner="administration",
+                retries=10,
+            ),
+            tags=["rollup"] + [f"label:{l}" for l in labels],
```

Comment:
Really, all this dag does is add some labeling so that the prometheus exporter 'picks up' on the generated XComs of tasks inside a rollup_dag.  See below.